drivers/overlay/overlay.go

```diff
@@ -1214,13 +1214,8 @@ func (d *Driver) recreateSymlinks() error {
 	if err != nil {
 		return fmt.Errorf("reading driver home directory %q: %v", d.home, err)
 	}
-	linksDir := filepath.Join(d.home, "l")
 	// This makes the link directory if it doesn't exist
-	rootUID, rootGID, err := idtools.GetRootUIDGID(d.uidMaps, d.gidMaps)
-	if err != nil {
-		return err
-	}
-	if err := idtools.MkdirAllAs(path.Join(d.home, linkDir), 0700, rootUID, rootGID); err != nil {
+	if err := idtools.MkdirAllAs(path.Join(d.home, linkDir), 0755, 0, 0); err != nil {
```

Comment:
Not blocking: Since commit is missing description for this change just a doubt previously this configured the remapped root uid/gid from provided uid and gid maps which is not same now i.e always `0/0` could this affect containers custom userns.